jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -150,13 +150,6 @@
           export OBS_CHECKOUT=$JHOME/OBS_CHECKOUT/$OBS_PROJECT
           export OSC_BUILD_ROOT=$JHOME/buildroot
 
-          mkdir -p ~/.obs
-          for i in incoming repo repourl
-          do
-            mkdir -p $JHOME/obscache/tar_scm/$i
-          done
```

Comment:
We don't need it afaics. The script just configures a different cache dir and uses that. And also creates the subdirs.